README.md

```diff
@@ -57,29 +57,33 @@ To run your R code across a cluster in Azure, we'll need to get keys and account
 ### Setup Azure Account
 First, set up your Azure Account ([Get started for free!](https://azure.microsoft.com/en-us/free/))
 
-Once you have an Azure account, you'll need to create the following two services in the Azure portal:
-- Azure Batch Account ([Create an Azure Batch Account in the Portal](https://docs.microsoft.com/en-us/azure/Batch/batch-account-create-portal))
-- Azure Storage Account (this can be created with the Batch Account)
+Once you have an Azure account, you'll need to create the following resources in Azure:
```

Comment:
Can we include a link to instructions for creating a free account?  For anyone new to Azure, this is a good, quick resource to show what you need and is what we link to in the Batch quickstarts: https://azure.microsoft.com/en-us/free/?ref=microsoft.com&utm_source=microsoft.com&utm_medium=docs&utm_campaign=visualstudio

Reply:
Isn't that was is already referenced on line 58?

Reply:
I must have missed that...